### PR TITLE
D0014: API finale

### PR DESCRIPTION
**Pull request by heinezen**

### CODE REVIEW

**Review by simonsan — APPROVED** [review]
Really nice work! 👍

File: content/blog/landing_page.md [hunk 1]
```diff
@@ -29,3 +29,4 @@ Summary: Landing page for all blog articles
 12. [Intermission I]({filename}/blog/D0011-openage_mod_api_updates.md)
 13. [Effects]({filename}/blog/D0012-openage_mod_api_effects.md)
 14. [Diplomatic Stances]({filename}/blog/D0013-openage_mod_api_diplo_stance.md)
+14. [Diplomatic Stances]({filename}/blog/D0014-openage_mod_api_finale.md)
```
Comment (TheJJ) [inline content/blog/landing_page.md]: copypasta error

File: content/blog/D0014-openage_mod_api_finale_0.1.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* In parallel, the engine will be extended to load API objects and implement their intended behavior
+* Once everything works, write a modding guide and tutorials
+
+Let's do it!
+
+## Fun facts
+
+As a bonus, here is nice collection of fun and pointless facts about the API and its development.
+
+* **The first API draft:**
```
Comment (TheJJ) [inline content/blog/D0014-openage_mod_api_finale_0.1.md]: i was confused about this at first, maybe it helps to write something like "our first API draft attempt (September 2008)" :grin:

the *first* api draft sounds like something we've done recently or is the result.

File: content/blog/D0014-openage_mod_api_finale_0.1.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+Title: D14: Openage modding API - Finale
+Date: 2019-05-31
+Tags: nyan, modding, API
+Authors: heinezen
+Summary: We look back at our API changes and summarize the new features
+
+Over the last year, we have introduced a lot of features to our API and spent a lot of time polishing and improving its core concepts. But as an old Chinese saying goes: "You have to let go, otherwise you will never finish the converter and jj will be unhappy". So today, we finally present you the first release version of the *openage Modding API* tree and will talk a little bit about the features that come with it.
+
+![modding API tree v0.1a]({static}/images/D0014-API-tree-0.1.svg)
```
Comment (TheJJ) [inline content/blog/D0014-openage_mod_api_finale_0.1.md]: i think we should add something like "we present you the result of hours of thought and discussions:" as a introduction to this image.

**Review by TheJJ — CHANGES_REQUESTED** [review]
awesome!

**Review by TheJJ — APPROVED** [review]

**Pull request merged**